src/lib/features/project-status/project-status-service.ts

```diff
@@ -35,14 +41,21 @@ export class ProjectStatusService {
             apiTokens,
             segments,
             activityCountByDate,
+            healthScores,
         ] = await Promise.all([
             this.projectStore.getConnectedEnvironmentCountForProject(projectId),
             this.projectStore.getMembersCountByProject(projectId),
             this.apiTokenStore.countProjectTokens(projectId),
             this.segmentStore.getProjectSegmentCount(projectId),
             this.eventStore.getProjectRecentEventActivity(projectId),
+            this.personalDashboardReadModel.getLatestHealthScores(projectId, 4),
         ]);
 
+        const averageHealth = healthScores.length
```

Comment:
This is a decision I made, in the personal dashboard, if you do not have 4 week data, it was showing 0. It felt silly so I made that in a "rare" case there is just 1-3 weeks, it will use that to calculate average. 

Althought the frontend will still show 4. But this is the first 3 weeks instance is set up. I think it makes sense to show average for 1-3 also, atleast I would like to see this chart already from week 1.

Reply:
Yea, that sounds sensible to me 😄 We could also send down either how many weeks of data we've used (so that it's correct) or calculate that in the front end based on the project's age 💁🏼